mesheryctl/internal/cli/root/perf/perf.go

```diff
@@ -136,14 +143,20 @@ func UpdateAuthDetails(filepath string) error {
 	return ioutil.WriteFile(filepath, data, os.ModePerm)
 }
 
-// perfCmd represents the Performance command
-var perfCmd = &cobra.Command{
+// PerfCmd represents the Performance command
+var PerfCmd = &cobra.Command{
 	Use:   "perf",
 	Short: "Performance Testing",
 	Long:  `Performance Testing & Benchmarking using Meshery CLI.`,
```

Comment:
Thinking better of this descriptor, could we change this to "Performance Management" - a broader term that foreshadows (and provides a landing spot) other capabilities to come forth.

```
Short: "Performance Management",
Long:  `Performance Management: Baselining and ongoing distributed stress, soak, and capacity testing of service mesh and workload responsiveness under load.`,
```